addon/mixins/actions-for-primitives/fd-creating-diagram-elements.js

```diff
@@ -3,7 +3,7 @@ import Ember from 'ember';
 /**
   Metods for creating joint js elements on diagrams.
 
-  @class FdCreatingDiagramElementsControllerMixin
+  @class FdCreatingDiagramElementsMixin
```

Comment:
These actions need to be performed inside the component, instead of transferring the `paper` from it to the outside.